src/assets/data/edition/series/1/section/5/op23/source-description.json

```diff
@@ -1548,7 +1548,7 @@
                 "pagination": "<em>- 1 -</em> bis <em>- 7 -</em>, <em>- 8 -</em> bis <em>- 12 -</em>, <em>- <s>13</s>&nbsp;<s>-</s> -</em> bis <em>- <s>14</s>&nbsp;<s>-</s> -</em>, <em>- 15 -</em> bis <em>- 16 -</em> auf Bl. 2<sup>r</sup>–5<sup>r</sup>, 6<sup>r</sup>–10<sup>r</sup> mit schwarzer Tinte.",
                 "measureNumbers": "Taktzahlen zu jedem Takt in I (<em>1</em> bis <em>50</em>), in II (<em>1</em> bis <em>30</em>) und in III (<em>1</em> bis <em>28</em>).",
                 "instrumentation": "",
-                "annotations": "<em>ARCHIV</em> | [darum im Kreis:] <em>DER UNIVERSAL-EDITION A. G. *</em> [alles in einem Kreis] auf Bl. 1–10 recto unten rechts mit violettem Stempelabdruck;<br /><em>10255</em> auf Bl. 1<sup>r</sup> unten Mitte mit Bleistift (nicht identifizierte Hs.);<br /><em>Der Dichterin zugeeignet</em> auf Bl. 1<sup>v</sup> System 5;</br />Dispositionsziffern in I nach T. 3, 6, 8, 10, 12, 14, 16, 18, 20, 23, 26, 29, 30 (<em><s>1</s></em> radiert), 32, 34 (<em><s>2</s></em> radiert), 35, 38, 41, 42 (<em><s>1</s></em> radiert), 44, 46 (<em><s>2</s></em> radiert), 47, 50;<br />in II nach T. 4 (<em><s>1</s></em> radiert), 5, 7 (<em><s>2</s></em> radiert), 9, 11 (<em>1</em>), 12, 13 (<em><s>2</s></em> radiert), 16, 19 (<em>4</em>), 20, 21 (<em>1</em>), 22, 23 (<em><s>2</s></em> radiert), 25, 26 (<em>3</em>), 30;<br /> in III nach T. 2 (<em>1</em>), 3, 4 (<em>2</em>), 5, 6 (<em>3</em>), 8, 10 (<em>2</em>), 11, 12 (<em>3</em>), 13, 14 (<em>4</em>), 15 (<em><s>4</s></em> radiert), 16, 17 (<em><s>1</s></em> radiert), 18 (<em>2</em>), 19, 20 (<em>3</em>), 21, 22 (<em>4</em>), 23, 24 (<em>1</em>), 26, 28<br /> mit Bleistift oder blauem Buntstift (nicht identifizierte Hs.).",
+                "annotations": "<em>ARCHIV</em> | [darum im Kreis:] <em>DER UNIVERSAL-EDITION A. G. *</em> [alles in einem Kreis] auf Bl. 1–10 recto unten rechts mit violettem Stempelabdruck;<br /><em>10255</em> auf Bl. 1<sup>r</sup> unten Mitte mit Bleistift (nicht identifizierte Hs.);<br /><em>Der Dichterin zugeeignet</em> auf Bl. 1<sup>v</sup> System 5;<br />Dispositionsziffern in I nach T. 3, 6, 8, 10, 12, 14, 16, 18, 20, 23, 26, 29, 30 (<em><s>1</s></em> radiert), 32, 34 (<em><s>2</s></em> radiert), 35, 38, 41, 42 (<em><s>1</s></em> radiert), 44, 46 (<em><s>2</s></em> radiert), 47, 50;<br />in II nach T. 4 (<em><s>1</s></em> radiert), 5, 7 (<em><s>2</s></em> radiert), 9, 11 (<em>1</em>), 12, 13 (<em><s>2</s></em> radiert), 16, 19 (<em>4</em>), 20, 21 (<em>1</em>), 22, 23 (<em><s>2</s></em> radiert), 25, 26 (<em>3</em>), 30;<br /> in III nach T. 2 (<em>1</em>), 3, 4 (<em>2</em>), 5, 6 (<em>3</em>), 8, 10 (<em>2</em>), 11, 12 (<em>3</em>), 13, 14 (<em>4</em>), 15 (<em><s>4</s></em> radiert), 16, 17 (<em><s>1</s></em> radiert), 18 (<em>2</em>), 19, 20 (<em>3</em>), 21, 22 (<em>4</em>), 23, 24 (<em>1</em>), 26, 28<br /> mit Bleistift oder blauem Buntstift (nicht identifizierte Hs.).",
```

Comment:
Finde den Unterschied nicht... :-( Sieht aber gut aus!

Reply:
`</br />Dispositionsziffern in I`

`<br />Dispositionsziffern in I`

War ein Slash zuviel

Reply:
👍